Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.xaml

```diff
@@ -3,28 +3,37 @@
                     xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
 
     <ResourceDictionary.ThemeDictionaries>
-        <ResourceDictionary x:Key="Default"/>
+        <ResourceDictionary x:Key="Default" />
         <ResourceDictionary x:Key="Light">
             <SolidColorBrush x:Key="MarkdownBorderBrush" Color="#FFBEBEBE" />
-            <SolidColorBrush x:Key="MarkdownBackgroundBrush" Color="#FFE0E0E0" />
+            <SolidColorBrush x:Key="MarkdownBackgroundBrush" Color="#FFF6F8FA" />
+            <SolidColorBrush x:Key="MarkdownInlineCodeBackgroundBrush" Color="#FFDDDFE1" />
+            <SolidColorBrush x:Key="MarkdownQuoteForeground" Color="#FF6E747C" />
         </ResourceDictionary>
         <ResourceDictionary x:Key="Dark">
```

Comment:
The Dark theme seems broken?

Reply:
You have to set the Foreground color in the Properties Panel, when switching to Dark mode, as it allows you to change it.